gr-filter/grc/filter_ival_decimator.block.yml

```diff
@@ -0,0 +1,34 @@
+id: ival_decimator
+label: IValue Stream Decimator
```

Comment:
If this is strictly for i values - perhaps the label should read "Interleaved Stream Decimator" to spell it out